src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/IAggregateValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    /// <summary>
+    /// Runs a set of <see cref="IValidator"/>s.
+    /// </summary>
+    public interface IAggregateValidator
+    {
+        /// <summary>
+        /// Human readable name of this <see cref="IAggregateValidator"/>.
+        /// </summary>
+        string Name { get; }
+
+        /// <summary>
+        /// Runs each <see cref="IValidator"/> and returns an <see cref="AggregateValidationResult"/> containing all results.
+        /// </summary>
+        Task<AggregateValidationResult> ValidateAsync(ValidationContext data);
+    }
+}
```

Comment:
@scottbommarito Looks good, just a couple of suggestions that might clean up the code a bit. Signing off though.